### ISSUE TITLE
Selector interpolation doesn't work properly with trailing comma.

### ISSUE THREAD

**Issue by cssmagic** [Stylus/bug#2]
> Node: v0.10.26
> Stylus: v0.49.3

## Passed

```css
$sel-multiple = '.a, .b,'
{$sel-multiple}
	display block
```

```css
.a,
.b {
  display: block;
}
```

#### Case 1

```css
$sel-multiple = '.a, .b,'
{$sel-multiple}
	span
		display block
```

```css
.a span,
.b span,
span {
  display: block;
}
```

#### Case 2

```css
$sel-multiple = '.a, .b,'
.wrapper
	{$sel-multiple}
		display block
```

```css
.wrapper .a,
.wrapper .b,
.wrapper {
  display: block;
}
```

**Comment by cssmagic** [comment]
I'm not sure if this is a bug, because `.a, .b` is not a valid selector in CSS.

if Stylus doesn't want to tolerate this, throwing an exception is OK.

**Comment by cssmagic** [comment]
Considered as known issue, maybe fix in 1.0. (LearnBoost/stylus # 1801)
